presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveBlockEncodingSerde.java

```diff
@@ -30,20 +31,35 @@
 import io.prestosql.spi.block.ShortArrayBlockEncoding;
 import io.prestosql.spi.block.SingleRowBlockEncoding;
 import io.prestosql.spi.block.VariableWidthBlockEncoding;
+import io.prestosql.spi.type.Type;
+import io.prestosql.spi.type.TypeId;
 
 import javax.inject.Inject;
 
+import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 
 import static com.google.common.base.Preconditions.checkArgument;
+import static io.prestosql.spi.type.BigintType.BIGINT;
+import static io.prestosql.spi.type.BooleanType.BOOLEAN;
+import static io.prestosql.spi.type.DateType.DATE;
+import static io.prestosql.spi.type.DoubleType.DOUBLE;
+import static io.prestosql.spi.type.HyperLogLogType.HYPER_LOG_LOG;
+import static io.prestosql.spi.type.IntegerType.INTEGER;
+import static io.prestosql.spi.type.TimestampType.TIMESTAMP_MILLIS;
+import static io.prestosql.spi.type.TimestampWithTimeZoneType.TIMESTAMP_WITH_TIME_ZONE;
+import static io.prestosql.spi.type.VarbinaryType.VARBINARY;
+import static io.prestosql.spi.type.VarcharType.VARCHAR;
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
```

Comment:
Why do we have `HiveBlockEncodingSerde`? What is it used for?

Reply:
funny 
```
// This class is exactly the same as BlockEncodingManager
public final class HiveBlockEncodingSerde
```

but the referenced class does not exist

Reply:
Seems to exist only for RecordingHiveMetastoreModule... I have no idea who added that, but it looks like @findepi was involved in the refactor that made it a top level class.

Reply:
did you mean me?